build.gradle

```diff
@@ -1,5 +1,6 @@
 buildscript {
-  repositories {
+    ext.kotlin_version = '1.3.72'
```

Comment:
can we move this here? https://github.com/segmentio/analytics-android/blob/master/build.gradle#L47

would make sense to have all the configs together